uber/config.py

```diff
@@ -429,3 +429,6 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
+# example output: 'MAGFest Feb 18 2016'
+c.EVENT_NAME_AND_DATE = c.EVENT_NAME + c.EPOCH.strftime(" %b %d %Y")
```

Comment:
You probably want ``%-d`` instead if ``%d``; the latter is zero padded so we'd end up saying ``Jan 03 20xx`` instead of what I imagine we'd want for an email subject, which would be ``Jan 3 20xx``.